### DISCUSSION TITLE
Payment for Copilot says it's successful but is ignored ?

### DISCUSSION THREAD

**Discussion by Mehdiway** [community/community#103292] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/1486876/5ee2efaf-f02e-4552-898f-c6c56764ad15)

When my subscription ended last month Github tried to take money from my account but failed 3 times because I had blocked my card. Now I'm trying to buy Copilot, I enter my billing details, it says the payment is successful! But Copilot is not working on my IDE. And when I go to check my payment history no payment is made. I tried 5 times now. Is it a bug ? Or is Github doing it on purpose since 3 withdrawals failed ?

**Comment by kgonzales1193** [comment]
Same. I says copilot is now ready to pair but still not able to use it. It seems it went through but isnt.
